base/src/test/java/io/spine/validate/RangeConstraintTest.java

```diff
@@ -59,6 +60,19 @@ void throwOnMalformedRanges(String badRange) {
         assertThrows(Exception.class, () -> RangeConstraint.rangeFromOption(badRange));
     }
 
+    @ParameterizedTest
+    @MethodSource("emptyRanges")
+    @DisplayName("throw on empty ranges")
+    void throwOnEmptyRanges(String emptyRange) {
+        assertThrows(IllegalArgumentException.class,
+                     () -> RangeConstraint.rangeFromOption(emptyRange));
+    }
+
+    private static Stream<Arguments> emptyRanges() {
+        return Stream.concat(rangeCombinationsFor(1, 0),
+                             Stream.of(Arguments.of("(0..0)")));
+    }
+
     private static Stream<Arguments> badRanges() {
```

Comment:
Why cannot `List` or `Set` be used instead of `Stream`? We have fixed set of values to check. We don't really stream.